lib/transformer.dart

```diff
@@ -98,64 +98,73 @@ class WebSkinDartTransformer extends Transformer implements LazyTransformer {
     TransformedSourceFile transformedFile = new TransformedSourceFile(sourceFile);
```

Comment:
Most of the diff in this file is just whitespace (indenting one level to be within the `else` block of the first check).